index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Span Registration Page</title>
+    <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
+  </head>
+  <body>
+    <div class="form-style-10">
+      <p class="title">
+        Microverse Entry Trial Survey!<span
+          >Sign up for the 6 - 8 months software training program</span
+        >
+      </p>
+
+      <form>
+        <div class="section"><span>1</span>Name, Email &amp; Age</div>
+        <div class="inner-wrap">
+          <label>Name<sup>**</sup> <input type="text" /> </label>
+          <label>Email<sup>**</sup> <input type="text" /></label>
+          <label>Age <input type="text" /></label>
+        </div>
+
+        <div class="section"><span>2</span>Other Details</div>
+        <div class="inner-wrap">
+          <label>
+            Which option best describes your current role?
+            <select>
+              <option>Select current role</option>
+              <option>Student</option>
+              <option>Full Time Job</option>
+            </select>
+          </label>
+
+          <label>Would you recommend freeCodeCamp to a friend? </label>
+          <p>
+            <label><input type="radio" name="option" /> Definitely</label>
+          </p>
+          <p>
+            <label><input type="radio" name="option" /> Maybe</label>
+          </p>
+          <p>
+            <label><input type="radio" name="option" /> Not Sure</label>
+          </p>
```

Comment:
- [ ] We should review how the overall code looks and try to maintain it as clean as possible. As you've noticed there are different ways of accomplishing the same thing. So how do we know which way is better? A rule of thumb is following this simple rule, keep it as simple, readable and clean as possible.

We shouldn't repeat the `<p>` tag that many times, and also is best practice to separate the `label` from the `inputs`. Here is a cleaner example of how we could improve our code:

```
<FORM action="http://somesite.com/prog/adduser" method="post">
    <P>
      <LABEL for="firstname">First name: </LABEL>
      <INPUT type="text" id="firstname"><BR>
      <LABEL for="lastname">Last name: </LABEL>
      <INPUT type="text" id="lastname"><BR>
      <LABEL for="email">email: </LABEL>
      <INPUT type="text" id="email"><BR>
      <INPUT type="radio" name="sex" value="Male"> Male<BR>
      <INPUT type="radio" name="sex" value="Female"> Female<BR>
      <INPUT type="submit" value="Send"> <INPUT type="reset">
    </P>
 </FORM>
```